setup.py

```diff
@@ -5,56 +5,124 @@
 
 from setuptools import setup, find_packages
 from unittest import TestLoader
+from setuptools.command.install import install
+from setuptools.command.develop import develop
+import subprocess
+import sys
+import os
+from shutil import copyfile, rmtree
+
+os.chdir(os.path.abspath(os.path.dirname(sys.argv[0]))) # needs to be in this directory!
+
+if sys.platform == 'win32':
+    START_FILE = 'StartLabGui.bat'
+    END_FILE = os.path.join('bin','LabGui.bat')
+else:
+    START_FILE = 'StartLabGui.sh'
+    END_FILE = os.path.join('bin','LabGui.sh')
+
+if 'install' not in sys.argv:
+    try:
+        os.mkdir('bin')
+    except: # already exists
+        pass
+
+    try:
+        with open(END_FILE, 'w+') as f:
+            f.close()
+    except:
+        print(sys.exc_info())
+    QUIET = True
+else:
+    QUIET = False
+
+
+def make_script():
+    # MAKE LAUNCH SCRIPT
+    try:
+        os.mkdir('bin')
+    except:
+        pass
+    copyfile(START_FILE, END_FILE)
 
 with open('README.rst') as readme_file:
     readme = readme_file.read()
 
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-requirements = ['Click>=6.0', ]
+requirements = ['Click>=6.0', 'virtualenv']
 
 setup_requirements = []
 
 test_requirements = []
 
+class PostDevelopCommand(develop):
+    """Post-installation for development mode."""
+    def run(self):
+        if QUIET:
+            subprocess.check_call([sys.executable, 'setup_new.py', 'quiet'])
+        else:
+            subprocess.check_call([sys.executable, 'setup_new.py'])
+        make_script()
+        develop.run(self)
+
+class PostInstallCommand(install):
+    """Post-installation for installation mode."""
+    def run(self):
+        if QUIET:
+            subprocess.check_call([sys.executable, 'setup_new.py', 'quiet'])
+        else:
+            subprocess.check_call([sys.executable, 'setup_new.py'])
+        make_script()
+        install.run(self)
 
 def test_suite():
     return TestLoader().discover('.')
 
 
 setup(
-    author="Bachibouzouk",
-    author_email='pfduc@physics.mcgill.ca',
+    author="zackorenberg",
+    author_email='zachary.berkson-korenberg@mail.mcgill.ca',
```

Comment:
Do you plan to do the long term support of LabGui? If not, then it would be better to leave my email. However the email should be `pierre-francois.duc@mail.mcgill.ca` as the other one will not be available in the future.

Reply:
I would like to, but ultimately it is your call.

Reply:
I am fine with it just remember to change it if you decide you don't want to be linked with LabGui anymore :)